myco_api_client/matching_algorithms/attributed_matching_algorithm.py

```diff
@@ -71,9 +71,9 @@ def _filter_offers_bids_by_attribute(
         for offer_bid in offers_bids:
             if attribute_key not in (offer_bid.get("attributes") or {}):
                 continue
-            if (isinstance(offer_bid.attributes.get(attribute_key), list)
-                    and attribute_value in offer_bid.attributes.get(attribute_key)
-                    or attribute_value == offer_bid.attributes.get(attribute_key)):
+            if (isinstance(offer_bid['attributes'].get(attribute_key), list)
```

Comment:
We tend to always use double quotes instead of single quotes unless it’s inevitable